lyricsgenius/api/api.py

```diff
@@ -181,6 +185,7 @@ def downvote_annotation(self, annotation_id, text_format=None):
         params = {'text_format': text_format or self.response_format}
         return self._make_request(path=endpoint, method='PUT', params_=params)
 
+    @check_token
```

Comment:
I didn't add the decorator to methods like `artist` or `song` because there is no way the user can access them unless they explicitly run this:
```
super(Genius, genius).artist(1665)
```
But if you'd like to add it to the rest of the `API` methods for consistency or other reasons, it won't be a problem.